substrate/frame/sudo/src/mock.rs

```diff
@@ -19,15 +19,12 @@
 
 use super::*;
 use crate as sudo;
-use frame_support::{derive_impl, traits::Contains};
-use sp_io;
-use sp_runtime::BuildStorage;
+use frame::testing_prelude::*;
 
 // Logger module to track execution.
-#[frame_support::pallet]
+#[frame::pallet]
 pub mod logger {
-	use frame_support::pallet_prelude::*;
-	use frame_system::pallet_prelude::*;
+	use frame::preliude::*
```

Comment:
```suggestion
	use frame::preliude::*;
```

Reply:
I can't believe I overlooked some minor changes. Thank you for pointing that out!🙏